Fabric.Identity.API/Services/ClaimsService.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Fabric.Identity.API.Configuration;
+using Fabric.Identity.API.Exceptions;
+using Fabric.Identity.API.Models;
+using Fabric.Identity.API.Quickstart.Account;
+using IdentityModel;
+using IdentityServer4.Models;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http.Authentication;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Security.Claims;
+
+namespace Fabric.Identity.API.Services
+{
+    public class ClaimsService

```

Comment:
would this be easier for testing if it had an interface?

Reply:
The controller is not the easiest to test, but this will help with that.  I can make that change.